lib/std/crypto.zig

```diff
@@ -93,6 +93,11 @@ pub const stream = struct {
     pub const ChaCha20With64BitNonce = @import("crypto/chacha20.zig").ChaCha20With64BitNonce;
 };
 
+/// Key derivation functions.
+pub const kdf = struct {
```

Comment:
Are these subnamespaces in random order? We should sort them alphabetically or by some other criteria

Reply:
I guess there are more or less in chronological order of addition.

But yeah, having them lexically sorted would make more sense. Done, thanks for suggesting this!